gRPC-Swift-Plugins.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
```

Comment:
This has been useful to see in the review but I don't think we should publish it to `master` yet (since the zip doesn't exist and no pod has been pushed).